styles.css

```diff
@@ -0,0 +1,72 @@
+*{
+    margin: 0;
+    
+}
+.app {
+    padding: 50px;
+    background: #1F2937;
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+    align-items: center;
+   
+}
+.header {
+    display: flex;
+    flex-direction: column;
+    gap: 10px; 
+    
+}
+h1{
+    font-family: "Karla",sans-serif;
+    font-weight: 800;
+    font-size: 40px;
+    line-height: 40px;
+    letter-spacing: -0.025em;
+}
+.top {
+    color: #FFFFFF;
+}
+.bottom {
+    color: #4ADF86;
+}
+p {
+    color: #D5D4D8;
+    font-family: "Inter",sans-serif;
+}
+button {
+    background: #10B981;
+    width: 190px;
+    height: 40px;
+    color: #FFFFFF;
+    font-family: "Inter",sans-serif;
+    box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.05);
+    border-radius: 6px;
+    cursor: pointer;
```

Comment:
-same here as i said above, try to use class name instead of using html elements.